openssl/src/sign.rs

```diff
@@ -283,6 +292,21 @@ impl<'a> Signer<'a> {
         }
     }
 
+    #[cfg(ossl111)]
+    pub fn len(&self) -> Result<usize, ErrorStack> {
```

Comment:
This method won't have its documentation anymore when building against 1.1.1.

Why does 1.1.1 need to call EVP_DigestSign and all other versions need to call DigestSignFinal?

Reply:
When used with EdDSA keys, `EVP_DigestSignFinal` fails with a segmentation fault
(even in `Signer::len()`). `EVP_DigestSign` supports all key types, but has only been available since 1.1.1.